### DISCUSSION TITLE
Welcome to DragonBallRedux Discussions!

### DISCUSSION THREAD

**Discussion by Oboe831** [Oboe831/DragonBallRedux#1] [General]
<h2><Strong>Welcome to Dragon Ball Redux!</Strong></h2>
Dragon Ball Redux is a fan project of a fan project, the amazing game the is Dragon Ball Universe (check it out at <a>https://dbu-rpg.com</a> if you haven't done so already).

This is a roll20 character sheet, that will be maintained and updated as regularly as I am able to! (I am a full-time student, so this will have to be worked around my Uni work).

If you come across any bugs (errors/mistakes/stuff that goes wrong/Gohan's development/etc) , just let me in the bug discussion section of the git and I'll try and remedy it better!

If you have any improvements or suggestions to the sheet, please put them in the ideas chat and I will have a look at their feasibility/whether they would break all my code!

Lastly, updates to the sheets may remove some/all data in a sheet, so please <h3>please</h3><h2>please</h2> save your data to a different location (.txt/word doc/google doc) before you update to the sheet!

**Comment by Jestyyy** [comment]
Any update on the reworked sheet you mentioned in the past?
